### PR TITLE
python docs section added

### LINKED ISSUE #524: [Feature Request]: Python docs addition
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

Adding multiple sections of Python documentation

1. Python data type
2. Python String
3. Python if...else and switch
4. Python loops

### Use Case

documentation of python

### Benefits

Adding Quality Documentation 
and also addig multiple sections(please consider it)

### Add ScreenShots

_No response_

### Priority

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I have starred the repository

### PR DESCRIPTION
**Pull request by Yashgabani845**
## Related Issue

[Feature Request]: Python docs addition  Fixes #524
### Topics
1. Datatypes
2. string
3. Conditional statement
4. Loops in python

## Type of PR

- [x] Feature enhancement
- [x] Documentation update

## Screenshots / Videos (if applicable)

https://github.com/CodeHarborHub/codeharborhub.github.io/assets/127788321/b1a770bc-bffb-4df0-9627-fb8f7d73ec76

[Attach any relevant screenshots or videos demonstrating the changes]

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Additional Context
i used official documentation and added quantity and quality content
kindly look over video and file changes 
and give level accordingly

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

File: docs/python/Strings.md [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+| `split()`       | Splits the string at the specified separator, and returns a list |
+| `splitlines()`  | Splits the string at line breaks and returns a list    |
+| `startswith()`  | Returns true if the string starts with the specified value |
+| `strip()`       | Returns a trimmed version of the string                |
+| `swapcase()`    | Swaps cases, lower case becomes upper case and vice versa |
+| `title()`       | Converts the first character of each word to upper case |
+| `translate()`   | Returns a translated string                            |
+| `upper()`       | Converts a string into upper case                      |
+| `zfill()`       | Fills the string with a specified number of 0 values at the beginning |
+```
```
Comment (Ajay-Dhangar) [inline docs/python/Strings.md:426]: remove <code>```</code>

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
update code based on suggestions

**Comment by Yashgabani845** [comment]
@Ajay-Dhangar done

**Review by Ajay-Dhangar — APPROVED** [review]
Done

**Pull request merged**